packages/core/test/unit/utils/utils.ts

```diff
@@ -18,36 +19,46 @@ function getNewDefaultVariableProcessor() {
   return DEFAULT_VARIABLE_PROCESSOR;
 }
 
-function getNewPluginManager(plugins, pluginsContext) {
-  const fileConfig = {
+export function getNewPluginManager(plugins: string[], pluginsContext: PluginContext) {
+  const fileConfig: NodeProcessorConfig = {
     baseUrlMap: new Set([ROOT_PATH]),
     baseUrl: '',
     rootPath: ROOT_PATH,
     outputPath: OUTPUT_PATH,
     headerIdMap: {},
     ignore: [],
     addressablePagesSource: [],
+    intrasiteLinkValidation: {
+      enabled: false, // TODO: true?
```

Comment:
What's this todo?

Reply:
Oversight on my part, thanks for the catch! `false` is the right default value.